automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainGroupSpec.scala

```diff
@@ -0,0 +1,224 @@
+                      }
+
+                      val workspaceSummaryPage = new WorkspaceSummaryPage(projectName, workspaceName).open
+                      checkWorkspaceFailure(workspaceSummaryPage, projectName, workspaceName)
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+
+      "when the user is inside of the group" - {
+        "when the workspace is shared with them" - {
+
+          "can be seen and is accessible" in {
+            val user = UserPool.chooseAuthDomainUser
+            implicit val authToken: AuthToken = authTokenDefault
+            withGroup("AuthDomain", List(user.email)) { authDomainName =>
+              withCleanBillingProject(defaultUser) { projectName =>
+                withWorkspace(projectName, "AuthDomainSpec_share", Set(authDomainName), List(AclEntry(user.email, WorkspaceAccessLevel.Reader))) { workspaceName =>
+                  withWebDriver { implicit driver =>
+                    withSignIn(user) { listPage =>
+                      val summaryPage = listPage.enterWorkspace(projectName, workspaceName)
+                      eventually {
+                        summaryPage.readAuthDomainGroups should include(authDomainName)
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+        "when the workspace is not shared with them" - {
+          "cannot be seen and is not accessible" in {
+            val user = UserPool.chooseAuthDomainUser
+            implicit val authToken: AuthToken = authTokenDefault
+            withGroup("AuthDomain", List(user.email)) { authDomainName =>
+              withCleanBillingProject(defaultUser) { projectName =>
+                withWorkspace(projectName, "AuthDomainSpec", Set(authDomainName)) { workspaceName =>
+                  withWebDriver { implicit driver =>
+                    withSignIn(user) { workspaceListPage =>
+                      eventually {
+                        workspaceListPage.hasWorkspace(projectName, workspaceName) shouldEqual false
+                      }
+
+                      val workspaceSummaryPage = new WorkspaceSummaryPage(projectName, workspaceName).open
+                      checkWorkspaceFailure(workspaceSummaryPage, projectName, workspaceName)
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+        //TCGA controlled access workspaces use-case
+        "when the workspace is shared with the group" - {
+          "can be seen and is accessible" in {
+            val user = UserPool.chooseAuthDomainUser
+            implicit val authToken: AuthToken = authTokenDefault
+            withGroup("AuthDomain", List(user.email)) { groupOneName =>
+              withGroup("AuthDomain", List(user.email)) { groupTwoName =>
```

Comment:
this is actually a multi-group test so it should go in the other spec

not your fault, this was mis-categorized before 😃